src/pdn/test/power_switch.api.tcl

```diff
@@ -0,0 +1,52 @@
+source "helpers.tcl"
+
+read_lef ../../../test/sky130hd/sky130hd.tlef 
+read_lef ../../../test/sky130hd/sky130_fd_sc_hd_merged.lef 
```

Comment:
Why did you delete the sky130hd link? Please restore and fix all the test to use that instead of `../../../test/`

Reply:
Ah - didnt catch on that this was a link. Fixed
Since the power_swtich.lef is a made up cell for the purposes of creating a testcase, I've moved this to the power_switch directory.